### PR TITLE
Refactor: Renames & Type Changes

### PR DESCRIPTION
**Pull request by gluax**
## Motivation

Unification of names and types across the `seda` ecosystem.

## Explanation of Changes

- [x] convert gas used and limit to u64
- [x] Renames:
  - [x] `dr_binary_id` -> `exec_program_id`
  - [x] `tally_binary_id` -> `tally_program_id`

## Testing

All tests still pass.

## Related PRs and Issues

N/A

### CODE REVIEW

File: src/msgs/data_requests/types.rs [hunk 1]
```diff
@@ -51,21 +51,21 @@ pub struct DataRequest {
...
+    pub tally_program_id:   String,
     /// Inputs for Tally WASM binary
     pub tally_inputs:       Bytes,
     /// Amount of required DR executors
     pub replication_factor: u16,
     /// Filter applied before tally execution
     pub consensus_filter:   Bytes,
     /// Amount of SEDA tokens per gas unit
-    pub gas_price:          U128,
+    pub gas_price:          u64,
```
Comment (Thomasvdam) [inline src/msgs/data_requests/types.rs]: This can/should stay a u128 right? Since it deals with `aseda`

Comment (mariocao) [reply src/msgs/data_requests/types.rs]: These kind of changes need to be updated in the evm contracts too, right?

Comment (Thomasvdam) [reply src/msgs/data_requests/types.rs]: I think it's literally everything that needs to change 😂

Comment (gluax) [reply src/msgs/data_requests/types.rs]: oops will revert this one my bad.

But yea EVM side needs to be updated as well I wrote a comment on the overlay side for that as well.

**Review by Thomasvdam — APPROVED** [review]

**Review by mariocao — APPROVED** [review]

**Pull request merged**